### ISSUE TITLE
Adding checkboxes AND ExcelTable onto a sheet results in corruption

### ISSUE THREAD

**Issue by kchristman54** [EPPlusSoftware/EPPlus#268]
We experimented with the new checkbox abilities that 5.5.0 introduces. While we could get them to work in a simple sheet, if a sheet has both an ExcelTable AND checkboxes, the file is corrupted.
![image](https://user-images.githubusercontent.com/33758481/103782440-14d8ba00-4ffd-11eb-8c75-a53aa8253a6c.png)

Here is some scratchcode that illustrates the problem. If you don't include the ExcelTable, then the checkboxes work fine. But if you include them, you get corrupted file.
```c#
        [Test]
        public void EpPlusFormControls()
        {
            string fileName = GetFileName("");
            FileInfo fileInfo = new FileInfo(fileName);
            ExcelPackage excelFile = new ExcelPackage(fileInfo);
            ExcelWorksheet formSheet = CreateFormSheet(excelFile);
            var r1 = formSheet.Drawings.AddCheckBoxControl("OptionSingleRoom");
            r1.Text = "Single Room";
            r1.LinkedCell = formSheet.Cells["G7"];
            r1.SetPosition(5, 0, 1, 0);
            var tableSheet = excelFile.Workbook.Worksheets.Add("Table");
            //ExcelRange tableRange = tableSheet.Cells[10, 20, 30, 22];
            //ExcelTable faultsTable = tableSheet.Tables.Add(tableRange, "FaultsTable");
            ExcelRange tableRange = formSheet.Cells[10, 20, 30, 22];
            ExcelTable faultsTable = formSheet.Tables.Add(tableRange, "FaultsTable");
            faultsTable.StyleName = "None";

            excelFile.Save();
        }
        private static ExcelWorksheet CreateFormSheet(ExcelPackage package)
        {
            var formSheet = package.Workbook.Worksheets.Add("Form");
            formSheet.Cells["A1"].Value = "Room booking";
            formSheet.Cells["A1"].Style.Font.Size = 18;
            formSheet.Cells["A1"].Style.Font.Bold = true;
            return formSheet;
        }
```

Thanks again from Texas A&M University's Energy Systems Lab!

**Comment by JanKallman** [comment]
The TablePart element is added before the Controls element, causing the worksheet xml file to violate the schema. 
This happends as the Controls element is inside an AlternateContent element. A fix will be provided shortly.

**Comment by JanKallman** [comment]
Fixed in version 5.5.1
